frontend/src/pages/Landing/index.css

```diff
@@ -110,14 +110,9 @@ h1.landing {
   border-color: #ECEEF1;
   border-bottom: none;
   cursor: pointer;
-  min-width: 90px;
```

Comment:
This made the checkbox columns wider than needed, and seemed unnecessary.

This also made me wonder if we should be using the USWDS classes, e.g. `.width-5`, either in HTML or if we should consider converting to Sass so we could build our custom css classes from the preexisting USWDS styles.